text/0000-simple_unwind_annotation.md

```diff
@@ -0,0 +1,75 @@
+- Feature Name: `simple_unwind_attribute`
+- Start Date: 2019-08-29
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+# Summary
+[summary]: #summary
+
+Provides an annotation to permit functions with explicit ABI specifications
+(such as `extern "C"`) to unwind
+
+# Motivation
+[motivation]: #motivation
+
+TODO
+
+- soundness & optimization
+- libjpeg/mozjpeg
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+TODO
```

Comment:
```suggestion
The `#[unwind(allowed)]` attribute permits functions with non-Rust ABIs (e.g. `extern "C" fn`) to unwind rather than terminating the process.
```

(The current verbiage is bad. Please suggest improvements.)

I don't think we've yet reached consensus on whether to include the `(allowed)`. Is it possible for annotations to exist in multiple forms, one that has an argument and one that doesn't? If not, I think we should use `(allowed)` for forwards compatibility.

Reply:
> Is it possible for annotations to exist in multiple forms, one that has an argument and one that doesn't?

Yes.